test/integration/testLoader.js

```diff
@@ -2,102 +2,199 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
-var assert = require('assert'),
+var request = require('request-promise'),
+    assert = require('assert'),
     util = require('./launchUtil'),
     path = require('path');
 
-describe('SERVER6 - duplicate service name should fail on startup', function () {
-    this.timeout(5000);
-
-    after(function (done) {
-        util.finish(done);
-    });
-
-    it('Launch server and check for failure', function (done) {
-        util.launch('server6', function(output) {
-            assert.ok(output.indexOf('already exists') > -1);
-            done();
+// describe('SERVER6 - duplicate service name should fail on startup', function () {
```

Comment:
was this exclusion meant to be temporary?

Reply:
Oops